usage.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: usage
+title: Quarkus Usage Analytics
```

Comment:
Shouldn't the title be `Quarkus Build Analytics`? or whatever we use in the main repo doc?

Reply:
What about: `Quarkus Build Analytics Usage` ?